### PR TITLE
ROC-5568 CCJ dashboard route tests

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5568

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/test/data/entity/ccjData.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export const ccjDeterminationByInstalment = {
+  ccjType: 'DETERMINATION',
+  paidAmount: 10,
+  paymentOption: 'INSTALMENTS',
+  repaymentPlan: {
+    paymentLength: '100 months',
+    completionDate: '2028-04-01',
+    paymentSchedule: 'EVERY_MONTH',
+    firstPaymentDate: '2020-01-01',
```
Comment (srineelesh) [inline src/test/data/entity/ccjData.ts]: can we not do something like this 'MomentFactory.currentDate().add(2, 'year')' as done below for payBySetDate

File: src/test/data/entity/courtDeterminationData.ts [hunk 2]
```diff
@@ -8,3 +8,35 @@ export const courtDeterminationData = {
...
+
+export const courtDeterminationChoseDefendantData = {
+  decisionType: 'DEFENDANT',
+  courtDecision: {
+    paymentOption: 'INSTALMENTS',
+    repaymentPlan: {
+      paymentLength: '100 months',
+      completionDate: '2028-04-01',
+      paymentSchedule: 'EVERY_MONTH',
+      firstPaymentDate: '2020-01-01',
```
Comment (srineelesh) [inline src/test/data/entity/courtDeterminationData.ts]: same as above

File: src/test/features/dashboard/routes/claimStatus/ccj.ts [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import { checkAuthorizationGuards } from 'test/features/dashboard/routes/checks/authorization-check'
+import { MomentFactory } from 'shared/momentFactory'
+import {
+  baseFullAdmissionData, basePayByInstalmentsData, basePayBySetDateData,
+  basePayImmediatelyDatePastData,
+  baseResponseData, partialAdmissionWithSoMPaymentBySetDateData
+} from '../../../../data/entity/responseData'
```
Comment (sabahirfan) [inline src/test/features/dashboard/routes/claimStatus/ccj.ts]: Would prefer not to use relative paths.

File: src/test/data/entity/ccjData.ts (hunk 1 above)
Comment (robertparkinson) [reply src/test/data/entity/ccjData.ts]: Sadly these dates can't be dynamic as the number of months will change day by day as we move through the calendar so I'd have to then write a function to figure out the number of months.

File: src/test/features/dashboard/routes/claimStatus/ccj.ts [hunk 4]
```diff
@@ -0,0 +1,590 @@
...
+import { PaymentOption } from 'claims/models/paymentOption'
+import {
+  courtDeterminationChoseClaimantData,
+  courtDeterminationChoseDefendantData
+} from '../../../../data/entity/courtDeterminationData'
```
Comment (sabahirfan) [inline src/test/features/dashboard/routes/claimStatus/ccj.ts]: please no relative path

File: src/test/features/dashboard/routes/claimStatus/ccj.ts [hunk 5]
```diff
@@ -0,0 +1,590 @@
...
+import {
+  ccjAdmissionBySpecifiedDate,
+  ccjDeterminationByInstalment,
+  ccjDeterminationBySpecifiedDate
+} from '../../../../data/entity/ccjData'
```
Comment (sabahirfan) [inline src/test/features/dashboard/routes/claimStatus/ccj.ts]: please no relative path

File: src/main/features/dashboard/views/macro/claimStatus/admissionByCCJDefendant.njk [hunk 6]
```diff
@@ -64,7 +79,7 @@
     {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
       <p>{{ t('They rejected your repayment plan.') }}</p>
     {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
-      <p>{{ t('They accepted your offer to pay {{ partAdmitAmount }}. They rejected your repayment plan.', { partAdmitAmount: claim.response.amount }) }}</p>
+      <p>{{ t('They accepted your offer to pay {{ partAdmitAmount }}. They rejected your repayment plan.', { partAdmitAmount: claim.response.amount | numeral }) }}</p>
```
Comment (andrewwa-kainos) [inline src/main/features/dashboard/views/macro/claimStatus/admissionByCCJDefendant.njk]: Does this fix [ROC-5904](https://tools.hmcts.net/jira/browse/ROC-5904)?

Comment (robertparkinson) [reply src/main/features/dashboard/views/macro/claimStatus/admissionByCCJDefendant.njk]: should do! i noticed it when I was adding the status and fixed it everywhere I saw it.

**Pull request merged**